package.json

```diff
@@ -17,9 +17,9 @@
     "lint:sol-fix": "npx prettier 'contracts/**/*.sol' --write && npx solhint 'contracts/**/*.sol' --fix",
     "lint": "npm run lint:ts && npm run lint:sol",
     "lint:fix": "npm run lint:ts-fix && npm run lint:sol-fix",
-    "postinstall": "husky install",
-    "prepack": "pinst --disable",
-    "postpack": "pinst --enable",
+    "_postinstall": "husky install",
```

Comment:
why is this prefixed?

Reply:
yes i noticed it's done by mistake. will update it

Reply:
resolved. merging